docs/en/ingest-management/fleet/fleet-server.asciidoc

```diff
@@ -1,38 +1,60 @@
 [[fleet-server]]
-= {fleet-server}
+= Set up {fleet-server}
 
-{fleet-server} is a component of the {stack} used to centrally manage {agent}s.
-It's launched as part of an {agent} on a host intended to act as a server. One
-{fleet-server} process can support many {agent} connections, and serves as a
-control plane for updating agent policies, collecting status information, and
-coordinating actions across {agent}s.
+{fleet-server} is required if you plan to use {fleet} for central management.
 
-{fleet-server} is the mechanism {agent}s use to communicate with {es}:
+* For self-managed deployments, you must install and host {fleet-server}
+yourself.
 
-. When a new agent policy is created, it's saved to {es}.
+* **No extra setup is required on {ecloud} unless you want to scale your
+deployment.** {ecloud} runs a hosted version of Integrations Server that
+includes {fleet-server}. 
+
+[discrete]
+== What is {fleet-server}?
+
+{fleet-server} is a component that connects {agent}s to {fleet}. It supports
+many {agent} connections and serves as a control plane for updating agent
+policies, collecting status information, and coordinating actions across
+{agent}s. It also provides a scalable architecture. As the size of your agent
+deployment grows, you can deploy additional {fleet-server}s to manage the
+increased workload.
+
+The following diagram shows how {agent}s communicate with {fleet-server} to
+retrieve agent policies:
+
+image:images/fleet-server-communication.png[{fleet-server} handles communication between {agent}, {fleet-server}, {fleet}, and {es}]
+
+. When a new agent policy is created, the {fleet} UI saves the policy to
+a {fleet} index in {es}.
 
 . To enroll in the policy, {agent}s send a request to {fleet-server},
 using the enrollment key generated for authentication.
 
-. {fleet-server} receives the request and gets the agent policy from {es},
-then ships the policy to all {agent}s enrolled in that policy.
+. {fleet-server} monitors {fleet} indices, picks up the new agent policy from
```

Comment:
From @blakerouse Mention the coordinator and that it also writes updated policies to the index.